planner/core/testdata/enforce_mpp_suite_in.json

```diff
@@ -132,5 +132,25 @@
       "EXPLAIN select count(distinct c+a), count(a) from t;",
       "EXPLAIN select sum(b), count(distinct c+a, b, e), count(a+b) from t;"
     ]
+  },
+  {
+    "name": "TestMPPMultiDistinct3Stage",
+    "cases": [
+      "EXPLAIN select count(distinct a) from t",
+      "select count(distinct a) from t",
+      "EXPLAIN select count(distinct a), count(distinct b) from t",
+      "select count(distinct a), count(distinct b) from t",
+      "EXPLAIN select count(distinct a), count(distinct b), count(c) from t",
```

Comment:
We need a test case where 
1. there are multiple distinct with a count(*).
2. there are multiple distinct with a avg(c+d).